addons/dispatch/1.2.x/dispatch-5.yaml

```diff
@@ -0,0 +1,110 @@
+# Do NOT edit this file directly.
```

Comment:
This should be updated in the Dispatch repository. https://github.com/mesosphere/dispatch/blob/dev/clusteraddon.yaml.tmpl

Reply:
whoops.. I'll update the template.